Leanplum-SDK/Classes/Features/Actions/LPActionManager.m

```diff
@@ -518,18 +518,6 @@ - (BOOL)hasTrackedDisplayed:(NSDictionary *)userInfo
     return NO;
 }
 
-- (void)trackMessageDisplayed:(NSDictionary *)userInfo
```

Comment:
@e7mac you sure this is an internal method?

Reply:
I approved it and then noticed Shiv has a question :).

Reply:
I approved it and then noticed Shiv has a question :).